### PR TITLE
Add contributing guidelines

### LINKED ISSUE #96: Add contributing guidelines
We have some good contributing guidelines in the FerretDB repo. We should copy relevant parts there and mention [github-actions-specific projects](https://github.com/FerretDB/github-actions/projects).

### PR DESCRIPTION
**Pull request by noisersup**
This PR closes #96.

### CODE REVIEW

**Comment by noisersup** [comment]
Whoops, I wanted to create a draft :worried:

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+This way, you will get help from us and avoid wasted efforts if something can't be worked on right now
+or someone is already working on it.
+
+You can find a list of good issues for first-time contributors [there](https://github.com/FerretDB/github-actions/contribute).
+
+## Setting up the environment
+
+### Requirements
+
+**TODO**
```
Comment (rumyantseva) [inline CONTRIBUTING.md:21]: @noisersup do you need any help with this part?

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+Fork the [FerretDB/github-actions repository on GitHub](https://github.com/FerretDB/github-actions/fork).
+To have all the tags in the repository and what they point to, copy all branches by removing checkmark for `copy the main branch only` before forking.
+
+After forking FerretDB on GitHub, you can clone the repository:
+
+```sh
+git clone git@github.com:<YOUR_GITHUB_USERNAME>/FerretDB.git
+cd FerretDB
+git remote add upstream https://github.com/FerretDB/FerretDB.git
+```
```
Comment (rumyantseva) [inline CONTRIBUTING.md:34]:
```suggestion
After forking `FerretDB/github-actions` on GitHub, you can clone the repository:

```sh
git clone git@github.com:<YOUR_GITHUB_USERNAME>/github-actions.git
cd github-actions
git remote add upstream https://github.com/FerretDB/github-actions.git
```
```

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+With `task` installed,
+you should install development tools with `task init`.
+
+If something does not work correctly,
+you can clean whole cache and re-install all development 
+tools with `task init-clean`.
+
+You can see all available `task` tasks with `task -l`.
+
+
```
Comment (rumyantseva) [inline CONTRIBUTING.md:54]: We need to add some info about testing - how to test an implemented change for GitHub Actions etc.

**Review by rumyantseva — CHANGES_REQUESTED** [review]
That's a good start, but we need to add more info!

File: CONTRIBUTING.md (hunk 1 above)
Comment (noisersup) [reply CONTRIBUTING.md:21]: Not now I believe

**Review by AlekSi — COMMENTED** [review]
Please re-request review when ready

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -0,0 +1,101 @@
...
+Above everything else, we value consistency in the source code.
+If you see some code that doesn't follow some best practice but is consistent,
+please keep it that way;
+but please also tell us about it, so we can improve all of it.
+If, on the other hand, you see code that is inconsistent without apparent reason (or comment),
+please improve it as you work on it.
+
+Our code most of the standard Go conventions,
+documented on [CodeReviewComments wiki page](https://github.com/golang/go/wiki/CodeReviewComments).
+Some of our idiosyncrasies:
```
Comment (AlekSi) [inline CONTRIBUTING.md:91]: The list is empty :) Let's remove that line

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -0,0 +1,101 @@
...
+
+## Contributing code
+
+### Commands for contributing code
+
+With `task` installed (see above), you may do the following:
+
+1. Format code with `task fmt`.
+2. Run linters against code with `task lint`.
+3. Run godocs server at 127.0.0.1:6060 to check documentation formatting.
```
Comment (AlekSi) [inline CONTRIBUTING.md:70]:
```suggestion
3. Run godocs server at 127.0.0.1:6060 to check documentation formatting with `task godocs`
```

For consistency

File: CONTRIBUTING.md (hunk 3 above)
Comment (AlekSi) [reply CONTRIBUTING.md:96]: ⚠️ That wasn't done – the document says nothing about running tests, GITHUB_TOKEN, etc.

File: README.md [hunk 6]
```diff
@@ -4,3 +4,12 @@ This repository contains our shared GitHub Actions, written in Go,
...
 [dance](https://github.com/FerretDB/dance),
 and other repositories.
+
+## Contributing
+
+* To contribute to this project, please read the [CONTRIBUTING.md](https://github.com/FerretDB/github-actions/blob/main/CONTRIBUTING.md) 
+and [CODE_OF_CONDUCT.md](https://github.com/FerretDB/FerretDB/blob/main/CODE_OF_CONDUCT.md) guideline to know more.
+* Is everything working as it should?
+If not, please let us know by [creating an issue](https://github.com/FerretDB/github-actions/issues/new/choose).
+You can create issues for bugs, documentation, features, or enhancements.
```
Comment (AlekSi) [inline README.md:14]: Unlike FerretDB, we don't have a wizard to select issue type, and external people can't assign labels

**Review by AlekSi — CHANGES_REQUESTED** [review]

File: CONTRIBUTING.md [hunk 7]
```diff
@@ -0,0 +1,100 @@
+# Contributing
+
+Thank you for your interest in making [FerretDB](https://github.com/FerretDB/FerretDB) better!
+
+## Finding something to work on
+
+We are interested in all contributions, big or small, in code or documentation.
+But unless you are fixing a very small issue like a typo,
+we kindly ask you first to [create an issue](https://github.com/FerretDB/github-actions/issues/new/choose),
```
Comment (w84thesun) [inline CONTRIBUTING.md:9]: That link should be fixed
```suggestion
we kindly ask you first to [create an issue](https://github.com/FerretDB/github-actions/issues/new),
```

**Review by w84thesun — CHANGES_REQUESTED** [review]

**Review by AlekSi — CHANGES_REQUESTED** [review]
Tests in internal/graphql/pull_request_test.go require `CONFORM_TOKEN` environemnt variable to be set. The value should be a personal access token with (I think) `read:org` and `read:project` permissions. We should:

* Add `test` Task target;
* Document it;
* Document the need to create a PAT

File: CONTRIBUTING.md [hunk 8]
```diff
@@ -0,0 +1,116 @@
...
+This way, you will get help from us and avoid wasted efforts if something can't be worked on right now
+or someone is already working on it.
+
+You can find a list of good issues for first-time contributors [there](https://github.com/FerretDB/github-actions/contribute).
+
+## Setting up the environment
+
+### Requirements
+
+The only way to run github-actions is to run them remotely on the repository.
```
Comment (rumyantseva) [inline CONTRIBUTING.md:21]:
```suggestion
The easiest way to run GitHub Actions is to run them remotely on the repository. However, we encourage everyone to write tests that can check the implemented functionality and can be run locally. 
```

File: CONTRIBUTING.md [hunk 9]
```diff
@@ -0,0 +1,116 @@
...
+
+You will need Go 1.19 or later on the host.
+If your package manager doesn't provide it yet,
+please install it from [go.dev](https://go.dev/dl/).
+
+You will also need `git` installed; the version provided by your package manager should do.
```
Comment (rumyantseva) [inline CONTRIBUTING.md:27]: I'm not sure we really need to mention git here... Like, people also need GitHub accounts to contribute, but we don't mention it :)

Comment (noisersup) [reply CONTRIBUTING.md:27]: It was inspired by the FerretDB's contributing guidelines

Comment (AlekSi) [reply CONTRIBUTING.md:27]: I think we actually added it because it wasn't clear for someone 😁

Comment (rumyantseva) [reply CONTRIBUTING.md:27]: Hmmm... Ok then :)

**Review by AlekSi — APPROVED** [review]

**Review by rumyantseva — APPROVED** [review]
LGTM

**Review by w84thesun — APPROVED** [review]
LGTM

**Pull request merged**